src/Cairo.jl

```diff
@@ -23,16 +23,16 @@ import Graphics: arc, clip, clip_preserve, close_path, creategc, device_to_user!
 import Base: copy, fill
 
 libcairo_version = VersionNumber(unsafe_string(
-      ccall((:cairo_version_string,Cairo.libcairo),Cstring,()) ))
+      ccall((:cairo_version_string,libcairo),Cstring,()) ))
 libpango_version = VersionNumber(unsafe_string(
-      ccall((:pango_version_string,Cairo.libpango),Cstring,()) ))
+      ccall((:pango_version_string,Pango_jll.libpango),Cstring,()) ))
 if !Sys.iswindows()
```

Comment:
Try removing this?  My hunch is that this was necessary because the wrong library was being referenced.